docusaurus/docs-internals/editor-controls/data-controls/fileUpload.md

```diff
@@ -176,3 +238,90 @@ In this case, the `fileUpload` control allows uploading only `.png,` `.jpg,` `.j
   }
 }
 ```
+
+#### CSS examples
+
+Show or hide the `.brz-download` element with CSS using a `multiSelect` control value.
+If `value.id` exist, then the color of `.brz-download` will be `green`;
+
+```js
+{
+  id: "file",
+  type: "fileUpload",
+  style: ({ value }) => {
+    if (value.id) {
+      return {
+        "{{WRAPPER}} .brz-download": {
+          color: "green"
+        }
+      }
+    }
+  }
+}
+```
+
+#### Usage in HTML example
+
+In the example below, we use the video output value to determine when to render the label in the button element.
+When the video is uploaded, we will render the label.
+We also use the same video values to add the `"data-disabled"` HTML attribute to the `.brz-button` and to create a `props.size` value for the `<Icon />` component.
+
+```tsx
+import { Brizy } from "@brizy/core";
+import { Icon } from "./Icon";
+import React, { JSX } from "react";
+
+interface Props {
+  file: {
+    name: string,
+    id: string
+  }
+}
+
+const Video = (props: Props): JSX.Element => {
+  const { name, id } = props.file;
+
+  const attributes = {
+    "data-disabled": id || name
+  }
+  
+  return (
+    <div className="brz-button" {...attributes}>
+      {id || name && <span>Delete</span>}
+      <Icon name="remove" size={id || name ? 16 : 24} />
+    </div>
+  );
+};
```

Comment:
i think we can make more simple example, so i propose to do an example to add a css class name when control return some valid value